content/docs/command-reference/remote/modify.md

```diff
@@ -406,7 +406,42 @@ application. To use a custom authentication method, use the following parameters
   ```dvc
   $ dvc remote modify myremote allow_anonymous_login true
   ```
+**Authentication examples:**
 
+Authentication example with `connection_string`:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify --local myremote connection_string 'mysecret'
+$ dvc remote push
+```
+  
+Authentication example with currently logged in az cli user/identity:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify myremote account_name 'myaccount'
+$ dvc remote push
+```
+  
+Authentication example with `account_key`:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify --local myremote account_name 'myaccount'
+$ dvc remote modify --local myremote account_key 'mysecret'
+$ dvc remote push
+```
+  
+Authentication example with `sas_token`:
+
+```dvc
+$ dvc remote add -d myremote azure://mycontainer/object
+$ dvc remote modify --local myremote account_name 'myaccount'
+$ dvc remote modify --local myremote sas_token 'mysecret'
+$ dvc remote push
+```  
+  
 Note that Azure remotes can also authenticate via environment variables (instead
```

Comment:
No `allow_anonymous_login` example?

Reply:
I could / did not test that either...

But I see your point: it does make sense to provide examples for all possible authentication scenarios. Should I add such examples even if I can not test them?